src/lib/LoginFile.js

```diff
@@ -78,17 +78,17 @@ class LoginFile {
         await loaded;
-        this._ctx.drawImage(img, 58, 43, 198.6, 24);
+        this._ctx.drawImage(img, 116, 86, 397.2, 48);
```

Comment:
Could use `this._drawDataUrlImage` here too?

Reply:
Oh, must have overlooked this. Thanks.